src/bootstrap/bootstrap.py

```diff
@@ -529,6 +534,30 @@ def build_triple(self):
 
         return "{}-{}".format(cputype, ostype)
 
+    def update_submodules(self):
+        if os.path.isdir(os.path.join(self.rust_root, ".git")) and \
+            (self.get_toml('submodules') != "false" and \
+                self.get_mk('CFG_DISABLE_MANAGE_SUBMODULES') != "1"):
+            print('Updating submodules')
```

Comment:
Could the indentation here be reduced by returning early with the conditions above?